src/export.js

```diff
@@ -49,11 +49,14 @@ if ( typeof window !== "undefined" ) {
 }
 
 // For nodejs
-if ( typeof module !== "undefined" && module.exports ) {
+if ( typeof module !== "undefined" && module && module.exports ) {
 	module.exports = QUnit;
+
+	// For consistency with CommonJS environments' exports
+	module.exports.QUnit = QUnit;
 }
 
 // For CommonJS with exports, but without module.exports, like Rhino
-if ( typeof exports !== "undefined" ) {
+if ( typeof exports !== "undefined" && exports ) {
```

Comment:
Double-checking what?  The first check (`typeof`) verifies that the variable has at least been declared, the second check verifies that its value is also truthy.

If I just did `if (exports)`, it would throw `ReferenceError`s... at least in oldIE. (e.g. http://stackoverflow.com/a/3390635/471696)